crates/bevy_text/src/font_atlas_set.rs

```diff
@@ -37,16 +43,27 @@ pub fn remove_dropped_font_atlas_sets(
     }
 }
 
+type FontSizeKey = u32;
```

Comment:
Nit: mild preference for a new-type rather than a type alias. It's a bit more misuse resistant.